Adding width to calving output

bearecinos opened the pull request:
Hello, 

Following the issue raised yesterday: https://github.com/OGGM/oggm/issues/875
I would like that in the glacier statistics function, we can also write the width of the calving front. I have now added a line in the inversion.find_inversion_calving() function so this variable can be included in the diagnostics.

Let me know if this is the way of solving this.
Thanks

pep8speaks commented:
Hello @bearecinos! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`oggm/core/inversion.py`](https://github.com/OGGM/oggm/blob/8a1ea7c94f6414f8e25c9dca7f6434a0e48ee1a3/oggm/core/inversion.py):

> [Line 442:1](https://github.com/OGGM/oggm/blob/8a1ea7c94f6414f8e25c9dca7f6434a0e48ee1a3/oggm/core/inversion.py#L442): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1

bearecinos commented:
@fmaussion  I have added the velocity function to the inversion.py and added that width parameter to the output of the new calving fucntion. Let me know if I need to change anything else so the tests pass.

fmaussion commented:
Thanks! For the sake of time I did some edits myself: note that I changed some names and that you now have to run `prepare_for_inversion` with `add_debug_var=True` for this to work. Please have a look and let me know.

bearecinos commented:
Thanks for merging !
